.github/dependabot.yml

```diff
@@ -3,7 +3,7 @@ updates:
   - package-ecosystem: "pip"
     directory: "/2.2"
     schedule:
-      interval: "weekly"
+      interval: "daily"
```

Comment:
I think the volume of these updates will be lower than those for GitHub Actions. Tightening the interval will also help ensure PRs get opened for dependency updates (e.g., Django) before we get to them.